packages/cloud-cognitive/src/internal/devtools.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { paramCase } from 'param-case';
+
+const devtoolsAttribute = 'data-carbon-devtools-id';
+
+function getDevtoolsId(componentName) {
+  return paramCase(componentName);
```

Comment:
I'm a fan of reusable packages, but adding param-case imposes this as a dependency for all users of the library. As we have a fairly simple case here -- we only need to convert our own component names (SomeKindOfThing or SomeKINDOfThing) -- we can perhaps eliminate the dependency at the cost of adding in a piece of slightly less general code. Something like:
```suggestion
  // convert to param-case (OneTwoTHREEFour -> one-two-three-four)
  componentName
    .replace(/([a-z])([A-Z])/g, '$1-$2') // OneTwoTHREE -> One-Two-THREE
    .replace(/([A-Z])([A-Z][a-z])/g, '$1-$2') // THREEFour -> THREE-Four
    .toLowerCase(); // One-Two-THREE-Four -> one-two-three-four
```

Reply:
I agree — I'm starting to wonder whether it's even necessary to change the case here, or if a `prefix` + `componentName` or `displayName` approach is sufficient.